src/compiler/parser-decl-symbtype.bas

```diff
@@ -393,6 +393,38 @@ function hIntegerTypeFromBitSize _
 
 end function
 
+private function cMangleModifier _
```

Comment:
Interesting, I like this idea. It could even be useful for some other things, like mangling to match C++ __int128.

Reply:
Cool.  I added a note to the source in cMangleModifier() to explain the choices so far and possible next steps.

> Interesting, I like this idea. It could even be useful for some other things, like mangling to match C++ __int128.

I just tried this little test on Win64 by allowing `alias "__int128"` to be attached to a struct (used an arbitrary mangle_dt number to map the mangling code 'o' for unsigned __int128):
```
unsigned long long int get128_lo( unsigned __int128 x )
{ return (unsigned long long int)x; }

unsigned long long int get128_hi( unsigned __int128 x )
{ return (unsigned long long int)(x >> 64); }
```
```
type uint128_t
	lo as ulongint
	hi as ulongint
end type

extern "c++"
	declare function get128_lo( byval x as uint128_t alias "__int128" ) as ulongint
	declare function get128_hi( byval x as uint128_t alias "__int128" ) as ulongint
end extern

dim x as uint128_t = ( &h12345678, &h87654321 )

print hex( get128_lo(x) )
print hex( get128_hi(x) )
```

And it worked!  Surely, I must be missing something... it's never that easy.